### PR TITLE
First draft for Problem Statement and Goals

### PR DESCRIPTION
**Pull request by lilweege**
# GitHub Ticket / Description

[SS-5](https://github.com/lilweege/SyntaxSentinels/issues/5)

## Type && Checklist

- [x] **New feature (new APIs, new functional methods)**
  - [ ] Created (or updated) unit test cases (if required)
  - [ ] Passed unit testing for **ALL** cases
  - [ ] Created (or updated) E2E cases (if required)
  - [ ] Passed E2E testing for **ALL** cases (local or/and sandbox)
- [ ] **Refactoring / Enhancement**
  - [ ] This is breaking change
  - [ ] Created (or updated) unit test cases (if required)
  - [ ] Passed unit testing for **ALL** cases
  - [ ] Created (or updated) E2E cases (if required)
  - [ ] Passed E2E testing for **ALL** cases (local or/and sandbox)
- [ ] **Bug-fix**
  - [ ] Type:
    - [ ] Code Issue:
    - [ ] Business Logic Issue:
  - [ ] Created (or updated) unit test cases (if required)
  - [ ] Passed unit testing for **ALL** cases
  - [ ] Created (or updated) E2E cases (if required)
  - [ ] Passed E2E testing for **ALL** cases (local or/and sandbox)

## Dependency

[Are there any required updates which cannot be done in this repo, or must be done after completing this PR?]

## Choose proper operation and replace the \<placeholder\>: `+semver:<placeholder>`

`breaking` `major` `feature` `minor` `fix` `patch`

+semver:patch

### CODE REVIEW

**Comment by lucasichen** [comment]
marking this as draft until doc is completed

**Comment by MohammadMohsinKhan** [comment]
Looks like there's a blank page on page 2. Goals look good to me except for the random extra content in the content which needs to be cleaned up.

**Review by lucasichen — CHANGES_REQUESTED** [review]

File: docs/ProblemStatementAndGoals/ProblemStatement.tex [hunk 1]
```diff
@@ -180,30 +203,37 @@ \section*{Appendix --- Reflection}
...
     did you resolve them?
 
     The main paint points came from gaining a grasp of the project and its
-    scope. The project itself is not simple by any standards. Figuring out what
-    we are doing and ensuring everyone on the team had the same understanding
-    was a challenge, which was resolved through a discussion. Aside from that,
-    there were no major paint points.
-
+    scope along with ascertaining what could be goals. The project itself is 
+    not simple  by any standards. Figuring out what we are doing and ensuring
```
[the remark this answers is not in the document]
Comment (N4T10N01) [reply docs/ProblemStatementAndGoals/ProblemStatement.tex:217]: I checked, it seems so. Now deleted.

**Review by lucasichen — APPROVED** [review]

**Review by MohammadMohsinKhan — COMMENTED** [review]
LGTM

**Review by MohammadMohsinKhan — APPROVED** [review]

**Pull request merged**